generators/view/index.js

```diff
@@ -29,15 +28,9 @@ var ViewGenerator = scriptBase.extend({
         this.jst_path = this.name + '-template';
       }
 
-      this._writeTemplate(
-        'view',
-        path.join(this.env.options.appPath + '/scripts/views', this.name),
-        {
-          appClassName: pascalCase(this.appname),
-          className: pascalCase(this.name),
-          'jst_path': this.jst_path
-        }
-      );
+      this._generate('view', {
+        'jst_path': this.jst_path
+      });
 
       if (!this.options.requirejs) {
```

Comment:
it's ok, script block for files should be added to index.html only if requirejs is not being used, I was implementing it also for test/index.html but the function to insert the script blocks is a bit fragile as it requires a good placeholder and it inserts before that rather than after and that made everything difficult as the file test/index.html comes from generator-mocha